README.md

```diff
@@ -10,3 +10,19 @@ Available at: [scanapi.dev](https://scanapi.dev)
 ## Tech Stack
 - [Jekyll](https://jekyllrb.com)
 - Deploys by [Netlify](https://www.netlify.com)
+
+## Introduction
+### Ruby installation
+- It is necessary to have ruby to run jekyll, to perform the verification with the following code, run `ruby -v`, if you return the version of ruby, you have it installed on your machine, otherwise install following the steps in documentation [here](https://www.ruby-lang.org/pt/downloads/)
```

Comment:
```suggestion
- It is necessary to have ruby to run jekyll, to perform the verification with the following code, run `ruby -v`, if you return the version of ruby, you have it installed on your machine, otherwise install following the steps in documentation [here](https://www.ruby-lang.org/en/downloads/)
```